BnbnavNetClient/Views/MapView.axaml.cs

```diff
@@ -19,6 +20,8 @@ public partial class MapView : UserControl
 {
     bool _pointerPressing;
     Point _pointerPrevPosition;
+    Point _viewVelocity;
+    List<Point> _pointerVelocities; // This list is averaged to get smooth panning.
```

Comment:
use ``readonly`` modifier as this won't be set again; initialize at definition (change to ``readonly List<Point> _pointerVelocities = new();``